### ISSUE TITLE
When I try to document incorrect-character-carriage-return, I keep getting a syntax-error

### ISSUE THREAD

**Issue by ollie-iterators** [PyCQA/pylint#8240]
### Bug description

This is the code that was used:
STRING = "
" # -1: [invalid-character-carriage-return]

This is the failed test for the code
FAILED doc/test_messages_documentation.py::test_code_examples[invalid-character-carriage-return-bad] - AssertionError: assert Counter({(2, ...-return'): 1}) == Counter({(2, ...x-error'): 1})

### Command used

```shell
pytest doc/test_messages_documentation.py
```

### Pylint output

```shell
FAILED doc/test_messages_documentation.py::test_code_examples[invalid-character-carriage-return-bad] - AssertionError: assert Counter({(2, ...-return'): 1}) == Counter({(2, ...x-error'): 1})
```

### Expected behavior

The error for the invalid-character-carriage-return is generated

### Pylint version

```shell
pylint 2.16.1
astroid 2.14.1
Python 3.10.2 (v3.10.2:a58ebcc701, Jan 13 2022, 14:50:16) [Clang 13.0.0 (clang-1300.0.29.30)]
```

**Comment by clavedeluna** [comment]
Hi @ollie-iterators I think your approach to opening an issue for every challenge you face isn't quite the right approach here. Issues are really meant to be for Pylint specific code issues. I'm going to close this one, so please move your question to either:
1. specifically add a comment in this issue https://github.com/PyCQA/pylint/issues/5953 since I believe that's what you're trying to contribute to
2. join the PyCQA discord or other channels to ask questions: https://meta.pycqa.org/getting-in-touch.html

**Comment by ollie-iterators** [comment]
Ok, I'll do that
